### PR TITLE
Update design of states #98

### PR DESCRIPTION
**Pull request by leeejune**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #98

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Added refactored state on radio.

Changes include:

- smaller button
- different colors for error state
- less padding for group radio buttons
- new focused state

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by leeejune** [comment]
I couldn't get husky to run the pre-commit hook. I got the same error from https://github.com/AlaskaAirlines/auro-radio/pull/99

**Comment by blackfalcon** [comment]
> I couldn't get husky to run the pre-commit hook. I got the same error from #99

What version of Node are you using? When using Node 16, I get the following error. 

```
husky > pre-commit (node v16.13.0)
Watching /Users/dalesande/src/auro/radio and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./node_modules ; is listed in .gitignore
Found & ignored ./coverage/Chrome Headless 102.0.5005.61 (Mac OS 10.15.7)/base.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 102.0.5005.61 (Mac OS 10.15.7)/prettify.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 105.0.5195.102 (Mac OS 10.15.7)/base.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 105.0.5195.102 (Mac OS 10.15.7)/prettify.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 111.0.5563.146 (Mac OS 10.15.7)/base.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 111.0.5563.146 (Mac OS 10.15.7)/prettify.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 91.0.4472.114 (Mac OS 10.15.7)/base.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 91.0.4472.114 (Mac OS 10.15.7)/prettify.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 94.0.4606.54 (Mac OS 10.15.7)/base.css ; has ignored extension
Found & ignored ./coverage/Chrome Headless 94.0.4606.54 (Mac OS 10.15.7)/prettify.css ; has ignored extension
Found & ignored ./demo/sass/style.css ; has ignored extension
Found & ignored ./src/auro-radio-group.css ; has ignored extension
Found & ignored ./src/auro-radio.css ; has ignored extension

Starting: test
node:internal/modules/cjs/loader:936
  throw err;
  ^
 ```
 
 I have to use Node 12 for this to work due to a really old version of Huskey that we need to update. Do this. 
 
 ```
 $ nvm use 12
 $ npm ci
```

File: src/auro-radio.scss [hunk 1]
```diff
@@ -106,23 +109,28 @@
...
-  margin-left: var(--auro-size-xl);
+  margin-left: calc(var(--auro-size-300));
+
+  &:hover:after {
+    background-color: var(--auro-color-ui-bkg-hover-on-light);
+  }
+
+  &:focus-visible:after {
+    border-color: var(--auro-color-border-primary-on-light);
+  }
```
Comment (blackfalcon) [inline src/auro-radio.scss:120]: This is the same update as the other PR?

File: src/auro-radio.scss [hunk 2]
```diff
@@ -106,23 +109,28 @@
...
 
   &:after {
     position: absolute;
     z-index: 1;
-    top: var(--auro-size-xxs);
-    left: var(--auro-size-xs);
+    top: var(--auro-size-50);
+    left: var(--auro-size-50);
```
Comment (blackfalcon) [inline src/auro-radio.scss:126]: These are intentional design changes?

File: src/auro-radio.scss [hunk 3]
```diff
@@ -106,23 +109,28 @@
...
 
-    width: var(--auro-size-lg);
-    height: var(--auro-size-lg);
+    width: calc(var(--auro-size-200) + var(--auro-size-50));
+    height: calc(var(--auro-size-200) + var(--auro-size-50));
```
Comment (blackfalcon) [inline src/auro-radio.scss:129]: These are intentional design changes?

File: src/auro-radio.scss [hunk 4]
```diff
@@ -106,23 +109,28 @@
...
 
     content: '';
 
     border-radius: 50%;
-  }
-
-  &:after {
```
Comment (blackfalcon) [inline src/auro-radio.scss:125]: The same change is on the other PR? Why was this removed?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: src/auro-radio.scss (hunk 4 above)
Comment (leeejune) [reply src/auro-radio.scss:125]: It was redundant.
